flocker/node/_docker.py

```diff
@@ -546,7 +546,7 @@ def _blocking_container_runs(self, container_name):
         ).write()
         return result['State']['Running']
 
-    def remove(self, unit_name):
+    def remove(self, unit_name, force=False):
```

Comment:
This method is an implementation of an interface method, `IDockerClient.remove`.

  * The interface needs to be updated similarly.
  * The other implementation needs to be updated similarly.
  * The interface-based test suite factory, `make_idockerclient_tests`, needs to have tests for the new functionality added.